python/hail/methods/impex.py

```diff
@@ -835,21 +913,23 @@ def import_gen(path,
            min_partitions=nullable(int),
            impute=bool,
            no_header=bool,
-           comment=nullable(str),
+           comment=oneof(str, sequenceof(str)),
            delimiter=str,
            missing=str,
            types=dictof(str, hail_type),
-           quote=nullable(char))
+           quote=nullable(char),
+           skip_blank_lines=bool)
 def import_table(paths,
                  key=[],
                  min_partitions=None,
                  impute=False,
                  no_header=False,
-                 comment=None,
+                 comment=[],
```

Comment:
use empty tuple for default arg

Reply:
now that we changed listof to sequenceof, this is better because lists are mutable.